### ISSUE TITLE
Installation: Tell user to create `package.json` file if does not exist

### ISSUE THREAD

**Issue by janpio** [prisma/prisma#2388]
When you follow just the `README.md` content in an empty folder, this happens:

```
C:\Users\Jan\Documents\throwaway\prismaReadme
λ npm install @prisma/client

> @prisma/client@2.0.0-beta.4 postinstall C:\Users\Jan\Documents\node_modules\@prisma\client
> node scripts/postinstall.js

warning In order to use "@prisma/client", please install @prisma/cli. You can install it with "npm add -D @prisma/cli".
npm WARN saveError ENOENT: no such file or directory, open 'C:\Users\Jan\Documents\package.json'
npm WARN enoent ENOENT: no such file or directory, open 'C:\Users\Jan\Documents\package.json'
npm WARN Documents No description
npm WARN Documents No repository field.
npm WARN Documents No README data
npm WARN Documents No license field.

+ @prisma/client@2.0.0-beta.4
added 1 package and audited 1 package in 7.734s
found 0 vulnerabilities


C:\Users\Jan\Documents\throwaway\prismaReadme
λ ls
```

Running the command from the warning has the same effect:

```
C:\Users\Jan\Documents\throwaway\prismaReadme
λ npm add -D @prisma/cli

> @prisma/cli@2.0.0-beta.4 preinstall C:\Users\Jan\Documents\node_modules\@prisma\cli
> node preinstall/index.js


> @prisma/cli@2.0.0-beta.4 install C:\Users\Jan\Documents\node_modules\@prisma\cli
> node download-build/index.js

npm WARN saveError ENOENT: no such file or directory, open 'C:\Users\Jan\Documents\package.json'
npm WARN enoent ENOENT: no such file or directory, open 'C:\Users\Jan\Documents\package.json'
npm WARN Documents No description
npm WARN Documents No repository field.
npm WARN Documents No README data
npm WARN Documents No license field.

+ @prisma/cli@2.0.0-beta.4
added 1 package from 1 contributor and audited 2 packages in 28.605s
found 0 vulnerabilities


C:\Users\Jan\Documents\throwaway\prismaReadme
λ ls
```

Only after you create a `package.json` you actually can install the CLI and later Client:

```
C:\Users\Jan\Documents\throwaway\prismaReadme
λ npm init -y
Wrote to C:\Users\Jan\Documents\throwaway\prismaReadme\package.json:

{
  "name": "prismaReadme",
  "version": "1.0.0",
  "description": "",
  "main": "index.js",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "keywords": [],
  "author": "",
  "license": "ISC"
}



C:\Users\Jan\Documents\throwaway\prismaReadme  (prismaReadme@1.0.0)
λ npm add -D @prisma/cli

> @prisma/cli@2.0.0-beta.4 preinstall C:\Users\Jan\Documents\throwaway\prismaReadme\node_modules\@prisma\cli
> node preinstall/index.js


> @prisma/cli@2.0.0-beta.4 install C:\Users\Jan\Documents\throwaway\prismaReadme\node_modules\@prisma\cli
> node download-build/index.js

npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN prismaReadme@1.0.0 No description
npm WARN prismaReadme@1.0.0 No repository field.

+ @prisma/cli@2.0.0-beta.4
added 1 package from 1 contributor and audited 1 package in 24.448s
found 0 vulnerabilities


C:\Users\Jan\Documents\throwaway\prismaReadme  (prismaReadme@1.0.0)
λ ls
node_modules/  package.json  package-lock.json
```

Similar to the warning in `@prisma/client` that it needs the CLI, we could output a message that the installation of the CLI requires an existing `package.json` file. It could look like this:
`warning In order to install "@prisma/cli", please create a package.json file. You can create it with "npm init -y".`

**Comment by abuuzayr** [comment]
I get the same warnings you did but my directory was not empty after `npm install @prisma/client`. Output of `ls` is 

```bash
node_modules            package-lock.json
```

Likewise after running `npm add -D @prisma/cli`, inside `node_modules/@prisma` are both `cli` and `client` folders.

```bash
  ~/Desktop/throw 
   npm add -D @prisma/cli   
> @prisma/cli@2.0.0-beta.4 preinstall /Users/fawyna/Desktop/throw/node_modules/@prisma/cli
> node preinstall/index.js


> @prisma/cli@2.0.0-beta.4 install /Users/fawyna/Desktop/throw/node_modules/@prisma/cli
> node download-build/index.js

npm WARN saveError ENOENT: no such file or directory, open '/Users/fawyna/Desktop/throw/package.json'
npm WARN enoent ENOENT: no such file or directory, open '/Users/fawyna/Desktop/throw/package.json'
npm WARN throw No description
npm WARN throw No repository field.
npm WARN throw No README data
npm WARN throw No license field.

+ @prisma/cli@2.0.0-beta.4
added 1 package from 1 contributor and audited 2 packages in 7.487s
found 0 vulnerabilities

  ~/Desktop/throw 
   ls    
node_modules            package-lock.json
```

I tried on both Mac OSX (Node v12.13.0) and Windows (v10.15.2) with the same results. Its probably good practice to have a `package.json` file anyway, so perhaps there should still be a warning?

**Comment by itsmingjie** [comment]
Hi! I'm one of the program coordinators of a nonprofit program called [CodeDay Init](https://labs.codeday.org/), and we partner with colleges to help CS students make their first open-source contribution. 

We'll claim this issue for now and a group of students will be on it soon with mentorship! If this issue is no longer relevant, please let me know.

**Comment by abuuzayr** [comment]
hey @itsmingjie ! Glad to have you here and happy to help students contribute. This issue may no longer be relevant, let me have a look and let you know soon.

**Comment by janpio** [comment]
Sorry, the "good first issue" should have definitely been removed with the answer from @abuuzayr because it is now unclear a) what the state is and b) what could be done to improve it. Might just be closed in the end.
